powerspikegg/rawdata/tools/fetch/summoner.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"io"
+	"errors"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/google/subcommands"
+
+	fetcherpb "powerspike.gg/powerspikegg/rawdata/fetcher/service_gopb"
+	lolpb "powerspike.gg/powerspikegg/rawdata/public/leagueoflegends_gopb"
+)
+
+type summonerCommand struct {
+	base
+
+	// flags
+	regionFlag string
+
+	// parsed flags / arguments
+	summonerName string
+	region lolpb.Region
+
+	// TODO(ArchangelX360): implement update by summonder ID
+}
+
+
+// registerSummonerCommand create a new summoner command
+// and registers it into subcommands
+func registerSummonerCommand() {
+	command := &summonerCommand{}
+
+	command.Initialize(
+		"summoner",
+		"Fetch summoner matches based on the summoner name",
+		"Query the rawdata fetcher in order to update a summoner " +
+			"and returns the list of updated matches. " +
+			"Use -region to change the default region.",
+	)
+
+	subcommands.Register(command, "")
+}
+
+
+func (c *summonerCommand) displaySummonerMatchResults(ctx context.Context,
```

Comment:
It is considered OK to not break this line (for some reasons...). I would suggest to do a one-liner (it makes it strangely more readable) or to break arguments on the method parenthesis. Here and on other methods.